charts/paperless/Chart.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: v2
+appVersion: 2.7.0
+description: Paperless - Index and archive all of your scanned paper documents
+name: paperless
+version: 0.1.0
+keywords:
+  - paperless
+home: https://github.com/k8s-at-home/charts/tree/master/charts/paperless
+icon: https://raw.githubusercontent.com/the-paperless-project/paperless/master/src/paperless/static/paperless/img/logo-dark.png
+sources:
+  - https://github.com/the-paperless-project/paperless
+maintainers:
+  - name:
+    email:
+dependencies:
+  - name: common
+    repository: https://k8s-at-home.com/charts/
+    version: ^1.6.3
```

```suggestion
    version: 1.6.3
```
The ^ character breaks our RenovateBot integration 🙁. Don't forget to update to the actual `common` version once it's published.